main.rb

```diff
@@ -113,13 +114,18 @@
   end
 end
 
-# add :pray: react to the God King (praise be btw) and :zachR: and :zachL: react to Toxic_Z
+# add :pray: react to the God King (praise be btw)
+# add :zachR: and :zachL: react to Toxic_Z
+# say "PRs welcome" when someone mentions making the bot/bishop do something
 bot.message do |event|
   event.message.react '🙏' if event.author.roles.any? { |r| r.id == CONFIG['roles']['god'] }
   if event.author.roles.any? { |r| r.id == CONFIG['roles']['toxic'] }
     event.message.react ':zachL:797961331101794344'
     event.message.react ':zachR:797961330929303583'
   end
+  if event.message =~ /make(?!.*\..).*?(bot|bishop)/i
+	event.channel.send_message('PRs welcome.')
```

Comment:
You should add the link to this repo to the message.

Reply:
I thought about it, my hesitation is discord embedding it every time and breaking up the conversation

Reply:
If the reply is an embed, the text can be formatted as a link as well.

Reply:
Switched to embeded response